src/connectors/numeric-refinement-list/connectNumericRefinementList.js

```diff
@@ -173,6 +172,85 @@ export default function connectNumericRefinementList(renderFn, unmountFn) {
         unmountFn();
         return state.clearRefinements(attributeName);
       },
+
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`